lua/roslyn/commands.lua

```diff
@@ -54,21 +83,12 @@ local subcommand_tbl = {
     },
     target = {
         impl = function()
-            local bufnr = vim.api.nvim_get_current_buf()
-            local root = vim.b.roslyn_root or require("roslyn.sln.utils").root(bufnr)
-
-            local roslyn_lsp = require("roslyn.lsp")
-
-            local targets = vim.iter({ root.solutions, root.solution_filters }):flatten():totable()
-            vim.ui.select(targets or {}, { prompt = "Select target solution: " }, function(file)
-                if not file then
-                    return
-                end
-
-                vim.lsp.stop_client(vim.lsp.get_clients({ name = "roslyn" }), true)
```

Comment:
Not 100% this is needed, but I think we can avoid changing it for this PR, then I can probably figure out at a later time

We still should stop the client when doing `Roslyn target`. Just not when doing `Roslyn start`